File: Dev/PythonServer/structure/map/vertex.py
class Vertex:
    def __init__(self):
        self.name = ''
        self.x = 0.0
        self.y = 0.0
    def parsing (self,value):
        if 'name' in value:
            self.name = value['name']
        if 'x' in value:
            self.x = value['x']
        if 'y' in value:
            self.y = value['y']
    def __str__(self):
        return "Name : %s; X : %s; Y : %s" % (self.name,self.x,self.y)

File: Dev/Android/Codes/Prod/MyApplication/app/src/main/java/com/example/quentin/myapplication/structures/map/Vertex.java
package com.example.quentin.myapplication.structures.map;

public class Vertex {

	private String name;
	private double x;
	private double y;
	
	// Default Constructor
	public Vertex(){
		this.name = "Default";
		this.x = 0.0;
		this.y = 0.0;
	}
	
	// Overloaded constructor
	public Vertex(String _name, double _x, double _y){
		this.name = _name;
		this.x = _x;
		this.y = _y;
	}

    @Override
    public String toString() {
        return "Vertex [name=" + name + ", x=" + x + ", y=" + y + "]";
    }


    // ****************** GETTERS ****************//

	public String getName() {
		return name;
	}

    public double getY() {
        return y;
    }

    public double getX() {
        return x;
    }

	// ***************** SETTERS ****************//

	public void setName(String _name) {
		this.name = _name;
	}

	public void setX(double x) {
		this.x = x;
	}

	public void setY(double y) {
		this.y = y;
	}
}
